scripts/build/testing.mk

```diff
@@ -5,8 +5,6 @@
 # Warning: make init-simapp will remove all data in simapp home directory
 #? init-simapp: Initializes a single local node network
 init-simapp:
-	./scripts/init-simapp.sh
```

Comment:
Can we instead say @echo use init-simapp-v2? That would break muscle memory when switching often branches between main and release/v0.52.x